api/handlers.go

```diff
@@ -28,6 +28,12 @@ import (
 )
 
 const (
+	// DefaultPageSizeSmall is the number of records we return when none is
+	// given and the objects are relatively large.
+	DefaultPageSizeSmall = 10
+	// DefaultPageSizeLarge is the number of records we return when none is
+	// given and the objects are relatively small.
```

Comment:
Follow Up: How is `relatively` defined? Might be helpful to add that context to this comment.